src/main/java/com/dev/theater/controller/CinemaHallController.java

```diff
@@ -33,8 +34,8 @@ public CinemaHallController(CinemaHallMapper cinemaHallMapper,
     }
 
     @PostMapping
-    public void addCinemaHall(CinemaHallResponseDto cinemaHallDto) {
-        CinemaHall cinemaHall = cinemaHallMapper.dtoToCinemaHall(cinemaHallDto);
+    public void addCinemaHall(CinemaHallRequestDto cinemaHallRequestDto) {
```

Comment:
let's test this method and see what may be missing  comparing to POST method in other controllers